### PR TITLE
Minor fix to Deep's description

### PR DESCRIPTION
**Pull request by roadrunner56**
**Bug fix**

## Summary
Adds a 'the' to the line `In the early days of hyperdrive`.

## Testing Done
I added 1 word.

### CODE REVIEW

**Comment by MasterOfGrey** [comment]
This isn't actually an error. This is a commonly used structure for phrasing regarding an era. i.e.:
- "In the early days of steam." -> when the steam engine was first revolutionising industry
- "In the age of sail." -> when generalising things relating to shipping prior to powered vessels

This is presented in that same vein.

**Review by LorenzoBolla — APPROVED** [review]

**Comment by LorenzoBolla** [comment]
> This isn't actually an error. This is a commonly used structure for phrasing regarding an era. i.e.:
> 
> * "In the early days of steam." -> when the steam engine was first revolutionising industry
> * "In the age of sail." -> when generalising things relating to shipping prior to powered vessels
> 
> This is presented in that same vein.

I'd argue it's more like
* "In the early days of boat"
* "In the early days of plane"

Which i'm  sure you can agree does not sound quite right. 

It could be "In the early days of hyperdrives" instead, too. Same as you'd phrase it as "In the early days of boats/planes".

**Comment by roadrunner56** [comment]
But no one says 'in the age of jet'. 'In the age of hyperdrive' sounds similar to me. 
The hyperdrive is a specific object, not a category of propulsion like steam engines or sailboats. 
Perhaps 'in the hyperdrive age' would be a good compromise?

**Comment by bene-dictator** [comment]
Mmmm – but yes, people do say Jet Age.

> You don't travel inside the hyperdrive, you travel inside a ship, that is propelled by a hyperdrive.

To expand on what Roadrunner said – by this logic, "the age of hyperdrive" should be equivalent to "the age of steam engine", if you were travelling by steam train or steamship. And "the age of steam engine" isn't correct, you'd preferably be saying "the age of the steam engine".

Another alternative might be `in the early days of hyperspace travel`. There is also a mention of `the early space age`, which might be better.

**Comment by roadrunner56** [comment]
I worry that `in the early days of hyperspace travel` might cause the description to overflow.

**Comment by xX-Dillinger-Xx** [comment]
I think you're all right. It all depends if you are using _**hyperdrive**_ as a noun or a verb. My terminology may not 100% accurate, I'm not an English major, but my point is valid. 
"The age of steam" uses steam as a verb.
 
Choices: 
**_In the early days of ....._**
- the hyperdrive 
  - if _hyperdrive_ is being used as a noun
- hyperdrive 
  - is also correct if _hyperdrive_ is being used as a verb
- hyperdrives
- hyperdrive travel
- space travel 

Here is a more relevant example. "overdrive"
"In the early days of overdrive" is correct. 
"In the early days of the overdrive" is also correct

**Comment by mOctave** [comment]
Steam, sail, and flight are all nouns of indeterminate quality. You could have a millilitre of steam or three hundred tons. Hyperdrive is not a noun of indeterminate quality: it is singular. If I say "hyperdrive" I mean a very specific single object.

You have to either mention a specific hyperdrive ("the early days of the hyperdrive"), pluralize the noun ("the early days of hyperdrives"), or attach a noun of indeterminate quantity ("the early days of hyperdrive travel") for it to work.

Or, any of bene or roadrunner's suggestions would work.

**Comment by MasterOfGrey** [comment]
The convention for this naming of things does (for some examples) explicitly violate conventional English grammar rules, so there's limited utility to arguing over the specific correctness - it's a terminology thing that is used by historians all on its own.

If people are really pressed about this, consider the following w.r.t to the convention:
1. Most Correct:
 a. "In the early days of hyperdrive."
2. Next most correct *drive* tagged option:
 a. "In the early days of the hyperdrive."
3. Less correct than 1, but *more* correct than 2:
 a. "In the early days of hypertravel." <- verb (equivalent to using "flight")
 b. "In the early days of hypertransit." <- verb/field (equivalent to using "aviation")
 c. "In the early days of hyperspace travel." <- note: reference to the media through which the verb occurs (Oceanic Navigation is associated with the Age of Discovery - a period distinct from previous maritime exploits.)

We should refrain from making reference to space age or space travel, as that would cover the period from when we had space travel within Sol, up until the discovery of the hyperdrive. (Alpha Centauri was settled during the "space age", the hyperdrive would have begun a new defined period after the space age.)

**Comment by xX-Dillinger-Xx** [comment]
> Hyperdrive is not a noun of indeterminate quality: it is singular. If I say "hyperdrive" I mean a very specific single object.
That is not entirely correct either. 
- " ..... go into hyperdrive." 
- it has hyperdrive 

It is not being used as a singular.

So I maintain my previous comment. "You are all correct"

**Comment by mOctave** [comment]
Here is a list of every way the word hyperdrive is currently used in game text, with some duplicates removed:
<details><summary><strong>Singular Noun:</strong></summary>
<ul>
<li>Chiron is the only planet outside the solar system that was colonized by humanity prior to the discovery of the hyperdrive</li>
<li>The original settlers arrived in the first century after the invention of the hyperdrive</li>
<li>Pelubta Station was built shortly after the Arachi discovered the hyperdrive</li>
<li>Given how much damage humans did to Earth before discovering the hyperdrive</li>
<li>when the invention of the hyperdrive was still recent</li>
<li>the construction of the first hyperdrive was still in living memory</li>
<li>Fighters do not come equipped with a hyperdrive</li>
<li>Or, you could press <Initiate hyperspace jump> to use your hyperdrive</li>
<li>Your hyperdrive can only travel along the &quot;links&quot; shown on your map</li>
<li>An ordinary hyperdrive jump consumes 100 fuel</li>
<li>This ship has no hyperdrive</li>
<li>its engines and hyperdrive</li>
<li>colonize nearby systems before the invention of the hyperdrive</li>
<li>he says they can be reached, and in turn reach us, with just a hyperdrive</li>
<li>he says they can be reached, and in turn reach humanity, with just a hyperdrive</li>
<li>We will give you a hyperdrive in its place</li>
<li>Okay, as long as you give me a hyperdrive to take its place</li>
<li>The Unfettered engineers quickly and carefully remove your jump drive and replace it with an ordinary hyperdrive</li>
<li>Once again, they swap your jump drive for a hyperdrive</li>
<li>the captain&#39;s flagship, a Shuttle that has seen better days, lacks a hyperdrive</li>
<li>Where&#39;s your hyperdrive</li>
<li>It&#39;s amazing that the hyperdrive was invented when it was too, because we could have easily wiped ourselves out if we hadn&#39;t</li>
<li>Earth before the hyperdrive was invented</li>
<li>it&#39;s impossible to get there that fast using a hyperdrive</li>
<li>A team of engineers meets you and quickly and efficiently removes your hyperdrive</li>
<li>This thing guzzles twice as much fuel per jump as an ordinary hyperdrive</li>
<li>If you&#39;ve got an outfitter on the planet where you&#39;re mining, do you need a hyperdrive, or even a shield generator</li>
<li>Before the hyperdrive</li>
<li>Charles Plot, father of the hyperdrive</li>
<li>Cleopatra lived closer in time to the invention of the hyperdrive than the building of the pyramids</li>
<li>They say that fewer than a dozen human beings alive today understand the inner workings of the hyperdrive</li>
<li>A scram drive is a specially designed hyperdrive</li>
<li>it has its own hyperdrive instead of needing to be carried inside a larger ship</li>
<li>They do not have any drives on board, not even a hyperdrive</li>
<li>squeeze a hyperdrive into otherwise intrasystem ships</li>
<li>Aren&#39;t you going to replace it with a hyperdrive</li>
</ul>
</details>

<details><summary><strong>Plural Noun:</strong></summary>
<ul>
<li>Dunites do not come equipped with nor have room for hyperdrives</li>
<li>It&#39;s just buried in there as a check from back in the days when hyperdrives would occasionally explode and we&#39;d need to work out who went missing</li>
<li>Reaching Elenchus would seem to be a problem, because it is behind systems we cannot reach with our hyperdrives</li>
<li>the dreadnoughts are lagging behind you because they are using ordinary hyperdrives</li>
<li>Activate your hyperdrives</li>
<li>we wouldn&#39;t even need hyperdrives</li>
<li>If that&#39;s true, then they would be able to visit the systems inaccessible to our hyperdrives</li>
<li>Hyperdrives were given to the government by the Quarg</li>
<li>Aliens took our people to the Deep before hyperdrives were invented to experiment on them</li>
<li>the internal workings bear some similarities to hyperdrives and Banishers</li>
<li>standard hyperdrives work by generating a fusion reaction</li>
<li>the new age of exploration opened by the advent of hyperdrives</li>
<li>before the discovery of the hyperdrive</li>
<li>it predates the discovery of the hyperdrive by, at minimum, five centuries</li>
<li>This ship has to have traveled between the stars for many of your years in order to reach this system without a hyperdrive</li>
<li>These include a hyperdrive, an enormous fusion reactor, and the most powerful plasma engines you&#39;ve ever seen</li>
</ul>
</details>

**Attributive Noun:**
- the station's gigantic ramscoops collect charged particles from the system's solar winds and refine them into hyperdrive fuel
- Most species we have encountered abandon any form of sub-light transportation between star systems immediately after developing hyperdrive technology
- inaccessible by hyperdrive

**Intentionally Bad Unfettered Hai Grammar:**
- But dragonfolk in the south had created hyperdrive
- we give you hyperdrive in place of jump drive

**Other:**
- more easily constructed and hyperdrive-capable transport ships are commonplace

Not sure what this one is supposed to mean. I think this is the only example in which hyperdrive is used as a verb in the whole game.

- _In the early days of hyperdrive the Kingsburg Corporation sent out what was then one of the largest and most technologically advanced colonization fleets_

As you can see, that's a grand total of one use of the word hyperdrive in a context where it acts as a verb, two with intentionally bad grammar, and then this one. ES does not use the phrase "go into hyperdrive" - the hyperdrive is a physical tangible thing that you use to go into hyperspace.

Sure, other games, movies, etc. use the word hyperdrive differently. But we aren't, and therefore this is still bad grammar.

**Comment by OcelotWalrus** [comment]
I don't think a "the" should be added to that sentence: if we put "the", it'd refer to the hyperdrive object/outfit, whereas without the "the", it refers to the hyperdrive systems/technology.

**Comment by mOctave** [comment]
My point above is that "hyperdrive systems/technology" aren't abbreviated to "hyperdrive" anywhere else. Maybe if it was "hyperdrive technology" in full it would work?

**Comment by OcelotWalrus** [comment]
For me, specifying "technology" isn't really needed. But yes, it'd work.

**Comment by MasterOfGrey** [comment]
> My point above is that "hyperdrive systems/technology" aren't abbreviated to "hyperdrive" anywhere else. 

That does not make it wrong.

**Comment by mOctave** [comment]
> > My point above is that "hyperdrive systems/technology" aren't abbreviated to "hyperdrive" anywhere else.
>
> That does not make it wrong.

It sort of does. Hyperdrive is only a very loosely defined word in any general dictionary. It's essentially a made-up word that we are defining based on how we use it. Our definitions of how space and FTL travel work are different from those of other sci-fi works. Our definition of hyperdrive is, too. We have no dictionary to work against, only what already exists in-game. I think we have to write our text so that, in order of priority, what we produce:
1. sounds natural,
2. fits ES' existing style and conventions,
3. fits conventions held elsewhere,
4. is realistic and believable.

Here, 4 doesn't really matter, because hyperdrives are fictional. But 1 and 2 do. I suppose 3 does, too, to some extent.

I would be happy with any of the following, because they fit (1) and (2):
- in the early days of the hyperdrive
- in the early hyperdrive age
- in the early galactic age
- in the early days of hyperdrives (although this sounds less fluid to me)
- in the early days of hyperdriving? (although that would involve making a new word)
- in the early days of deuterium fuel (although that would be confusing)

However, my favourite is "in the early days of hyperspace travel." I've checked, and it works just fine without overflowing.

Screenshot 2024-04-17 at 6 29 42 PM

> The convention for this naming of things does (for some examples) explicitly violate conventional English grammar rules, so there's limited utility to arguing over the specific correctness - it's a terminology thing that is used by historians all on its own.

I haven't really addressed this, but I have yet to find a single example where it does actually violate grammar rules, either in your examples or in my own limited research. Could you provide one?

**Comment by MasterOfGrey** [comment]
> I haven't really addressed this, but I have yet to find a single example where it does actually violate grammar rules, either in your examples or in my own limited research. Could you provide one?

If you interpret it as the technology it doesn't - if you interpret it in the ways all those above were talking about it, it does, and the comment was in context of that.

> However, my favourite is "in the early days of hyperspace travel." I've checked, and it works just fine without overflowing.

Given the way people use words and language, I would expect this to get condensed fairly quickly into something like "hypertravel" or "hypertransit".
If this is the route we go, I would suggest that settling on a term like that of our own choosing rather than the expanded form.

**Comment by mOctave** [comment]
> Given the way people use words and language, I would expect this to get condensed fairly quickly into something like "hypertravel" or "hypertransit".
> If this is the route we go, I would suggest settling on a term like that of our own choosing rather than the expanded form.

Maybe, but we're already using the words "hyperspace travel" in several places throughout the game.

**Comment by xX-Dillinger-Xx** [comment]
I suggest just leaving as it was. It is grammatically correct, and lets be honest the only people that even noticed or care are involved in this conversation. I don't even remember reading the description of the deep, let alone that it "might" of been missing a "the".

Hyperdrive would be used the same way that overdrive is used.  _"In the early days of overdrive"_ is grammatically correct and is used that way.

File: data/map planets.txt [hunk 1]
```diff
@@ -880,7 +880,7 @@ planet Deep
...
-	description `	The fishing settlements are actually the second colonization attempt on this planet. In the early days of hyperdrive the Kingsburg Corporation sent out what was then one of the largest and most technologically advanced colonization fleets, penetrating deep into the then-unexplored southern galactic rim. After two centuries of sporadic and guarded contact, the colony fell silent, and when human expansion finally reached this area again, no trace of the original colony was found.`
+	description `	The fishing settlements are actually the second colonization attempt on this planet. In the early days of the hyperdrive the Kingsburg Corporation sent out what was then one of the largest and most technologically advanced colonization fleets, penetrating deep into the then-unexplored southern galactic rim. After two centuries of sporadic and guarded contact, the colony fell silent, and when human expansion finally reached this area again, no trace of the original colony was found.`
```
Comment (Anarchist2) [inline data/map planets.txt:883]:
```suggestion
	description `	The fishing settlements are actually the second colonization attempt on this planet. In the early days of hyperspace travel the Kingsburg Corporation sent out what was then one of the largest and most technologically advanced colonization fleets, penetrating deep into the then-unexplored southern galactic rim. After two centuries of sporadic and guarded contact, the colony fell silent, and when human expansion finally reached this area again, no trace of the original colony was found.`
```

**Review by Anarchist2 — COMMENTED** [review]
I don't think the current description works. The closest comparison to the hyperdrive for me is something like "combustion engine", and "early days of the combustion engine" is a lot more common than "early days of combustion engine".

**Pull request merged**